.kitchen.docker.yml

```diff
@@ -29,4 +29,5 @@ suites:
       inspec_tests:
         - name: chef-aps-inspec-tomcat
           git: https://github.com/Alfresco/chef-aps-inspec-tomcat
+          branch: 'TAA-930'
```

Comment:
The PR for the chef-aps-inspec-tomcat is not merged yet. otherwise it will use the master branch and will fail